conftest.py

```diff
@@ -0,0 +1,102 @@
+
+import pytest
+import requests_mock
+
```

Comment:
Are you shure, that this file should be stored in project root? 
And add module's description

Reply:
Contest in the root directory is a best practice to help us avoid adding init files to the test directory. It facilities local imports, especially as they pertain to testing.

https://stackoverflow.com/questions/34466027/in-pytest-what-is-the-use-of-conftest-py-files

> Test root path: This is a bit of a hidden feature. By defining conftest.py in your root path, you will have pytest recognizing your application modules without specifying PYTHONPATH. In the background, py.test modifies your sys.path by including all submodules which are found from the root path.